From 018356ae36827f10ee5a4baaac1f0b4dce8e451e Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Wed, 5 Feb 2025 16:52:33 +0100
Subject: [PATCH 1583/1695] dts: stm32: drop wakeup-parent from tamp nodes

The property is not used anymore.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git ab16138abf7dc4eb50db4880dbf5cd0932007c5a dts: stm32: drop wakeup-parent from tamp nodes]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: Ie0c1bdc605bc3a75c10e3857e37568b2edf7f43e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/434929
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts  | 1 -
 core/arch/arm/dts/stm32mp235f-dk.dts  | 1 -
 core/arch/arm/dts/stm32mp257f-dk.dts  | 1 -
 core/arch/arm/dts/stm32mp257f-ev1.dts | 1 -
 4 files changed, 4 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index f15b93a6d..94531dcc8 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -686,7 +686,6 @@
 	st,tamp-passive-nb-sample = <4>;
 	st,tamp-passive-sample-clk-div = <16384>;
 	wakeup-source;
-	wakeup-parent = <&exti 0xffff 0>;
 
 	/* Tamper button */
 	tamp-button {
diff --git a/core/arch/arm/dts/stm32mp235f-dk.dts b/core/arch/arm/dts/stm32mp235f-dk.dts
index 0fc1c67f5..ad5864741 100644
--- a/core/arch/arm/dts/stm32mp235f-dk.dts
+++ b/core/arch/arm/dts/stm32mp235f-dk.dts
@@ -454,7 +454,6 @@
 
 &tamp {
 	wakeup-source;
-	wakeup-parent = <&exti2 0xffff 0>;
 	st,tamp-passive-precharge = <2>;
 	st,tamp-passive-nb-sample = <4>;
 	st,tamp-passive-sample-clk-div = <16384>;
diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index b90d6f7e5..aab9c8800 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -458,7 +458,6 @@
 
 &tamp {
 	wakeup-source;
-	wakeup-parent = <&exti2 0xffff 0>;
 	st,tamp-passive-precharge = <2>;
 	st,tamp-passive-nb-sample = <4>;
 	st,tamp-passive-sample-clk-div = <16384>;
diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 860b30c02..677a5ec41 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -490,7 +490,6 @@
 
 &tamp {
 	wakeup-source;
-	wakeup-parent = <&exti2 0xffff 0>;
 	st,tamp-passive-precharge = <2>;
 	st,tamp-passive-nb-sample = <4>;
 	st,tamp-passive-sample-clk-div = <16384>;
-- 
2.47.2

